label_studio/users/templates/users/user_account.html

```diff
@@ -233,7 +233,6 @@
       <table>
         <tr><td style="{% if user.allow_newsletters is None %}border: 1px red solid; border-radius: 5px{% endif %}">
 
-          <input name="email" type="hidden" value="{{ user.email }}">
           <input name="first_name" type="hidden" value="{{ user.first_name }}">
           <input name="last_name" type="hidden" value="{{ user.last_name }}">
```

Comment:
Nice find, wondering if we need to send first_name and last_name also.

Reply:
I don't think they are necessary either, but we will be taking this page to React this sprint, so we'll remove this pretty soon.

Reply:
That was my thoughts, its a PATCH to the API so typically speaking no. It should only send what is changing.

Reply:
agreed, I'll remove them